packages/types/src/payment-types.ts

```diff
@@ -86,6 +87,7 @@ export interface IPaymentNetworkEvent<TEventParameters> {
   name: EVENTS_NAMES;
   parameters?: TEventParameters;
   timestamp?: number;
+  from?: IIdentity;
```

Comment:
this will be used only by declarative, right? 

I think adding a comment is required here

Reply:
Only for declarative for now. But could be use somewhere else if needed later.